Backend/Data_/Dtos/UserCreateDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Data_.Entities;
```

Comment:
You can use only one DTO per entity. There is no need to use separate DTO for Update and Create actions because they are pretty the same. So please create one UserDTO and AchievementDTO.